webnn/resources/utils.js

```diff
@@ -660,10 +661,11 @@ const buildPad = (operationName, builder, resources) => {
 };
 
 const buildReshape = (operationName, builder, resources) => {
-  // MLOperand reshape(MLOperand input, sequence<unsigned long?> newShape);
+  // MLOperand reshape(MLOperand input, sequence<unsigned long> newShape);
```

Comment:
Hmm, what does this commented line mean? Does it mean you're implementing that API function here? If so, suggest adding that to the comment:

```
Implements:
MLOperand reshape(MLOperand input, sequence<unsigned long> newShape);
MLOperand reshape(MLOperand input, sequence<unsigned long> newShape);
```

Reply:
I removed '**?**' character since [Chromium CL#5060264](https://chromium-review.googlesource.com/c/chromium/src/+/5060264) - Drop null value support in newShape parameter for **reshape** has already been merged, link to https://github.com/webmachinelearning/webnn/issues/388.

Reply:
Aha.